src/test/country.test.js

```diff
@@ -0,0 +1,156 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import * as reactRedux from 'react-redux';
+import Home from '../mock/home';
+
+jest.mock('react-redux');
```

Comment:
Great job @The-Normal-One1 🤝

- [ ] [Optional] I would recommend you add more tests for the details and NotMatch components.